cli/import.full.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
can we put these import configuration files to testing repo? I think the configuration files are better to put together with the data.

Reply:
Sure. I will move it.